spec/lib/recog/nizer_spec.rb

```diff
@@ -28,6 +28,33 @@
     end
   end
 
+  describe ".multi_match" do
```

Comment:
Regardless of whether or not you just collapse this into `match`, you should test the negative case where there are no matches.